server/player.lua

```diff
@@ -744,6 +747,7 @@ function CreatePlayer(playerData, Offline)
             TriggerClientEvent('hud:client:OnMoneyChange', self.PlayerData.source, moneytype, amount, false)
             TriggerClientEvent('QBCore:Client:OnMoneyChange', self.PlayerData.source, moneytype, amount, 'add', reason)
             TriggerEvent('QBCore:Server:OnMoneyChange', self.PlayerData.source, moneytype, amount, 'add', reason)
+            TriggerEvent('qbx_core:server:setMoney', self.PlayerData.source, moneytype, self.PlayerData.money[moneytype])
```

Comment:
Why the new event?

Reply:
To make sure qb users aren't able to use the ox_inventory bridge like linden asked

Reply:
Should we call this new event onMoneyChange as well and pass it old amount, new amount, and reason?

Reply:
refactored to separate event and calls ox_inventory export directly